modules/fedimint-ln/src/lib.rs

```diff
@@ -224,7 +226,21 @@ pub struct LightningVerificationCache;
 pub struct LightningModuleConfigGen;
 
 #[async_trait]
-impl FederationModuleConfigGen for LightningModuleConfigGen {
+impl ModuleInit for LightningModuleConfigGen {
```

Comment:
Should we also rename `LightningModuleConfigGen` to `LightningModuleInit` (if we keep the module init name)?

Reply:
Yes. I was just in a rush to see if I can get it work.

Reply:
Rename cleanups in another PR.

Reply:
https://github.com/fedimint/fedimint/issues/1232